### ISSUE TITLE
Multiple layers

### ISSUE THREAD

**Issue by tomnolan** [mjungnickel18/papyruscs#33]
Is it currently possible with the app, or are there plans to implement a way, to drill down to layers below the top level of the ground to explore underneath?

**Comment by clarkx86** [comment]
Hey @tomnolan :-)!
It is already possible to set a Y-level threshold with `-y` or `--limity`.

**Comment by tomnolan** [comment]
A few things about that...

1. y/limity seems to be just to limit the top layer that is displayed in the rendered map. What I'm really referring to would be to be able to have the map have a drop down list of y heights that allows you to step through every y level of the map to see every block at that level.
2. http://papyrus.gwsa.de/ shows Overworld, Underground, Nether, The End. My request is really to expand the functionality of the Underground profile.
3. As a follow-up to #2, it seems the profile parameter only accepts a string. How do you specify an array to generate all profiles for all dimensions like seen on your sample site?

**Comment by karpana** [comment]
@tomnolan,

I just want to claify, for my own personal benefit, what you mean by expand?
Do you mean provide multiple sub-layer points where you can see the underground at say layer 40, and another at layer 30, 20 and so forth, almost like flipping through layers?

**Comment by tomnolan** [comment]
Exactly, @karpana . Mcpe_viz/bedrock_viz do this and so do a number of similar tools for Java. Basically you get every y level as a separate set of images. I'm not sure how this would work with the way papyrus handles the render strategy, I think an entirely different method would be necessary to make this work properly.

**Comment by danobri** [comment]
Related to this - the documentation on the -ylimit parameter is misleading / confusing:

> -y, --limity                    (Default: -1) Limits the chunk rendering in the y dimension
                                  (inclusive). For y provide just one positive value, eg: 10. -1 means:
                                  all

If the application can only generate and display a map for a single Y value, then the default of -1 does not mean "all", like the documentation says, right?  From what I can tell the default is actually sea level (62), correct?

**Comment by karpana** [comment]
I think the interpration of -1 means, not "sea level" but the highest level visible from the maximum height.
In effect, that typically means "sea height or higher", but, as you can see if you hollow out a space in the sea level and drain it, you'll see the hollowed out space that is below sea level.  Or if you build a hole all the way to bedrock, you'll see bedrock in your generated view.
